requirements.txt

```diff
@@ -13,3 +13,7 @@ torchsde==0.2.6
 einops
 opencv-contrib-python
 safetensors
+
+xformers
+sageattention
+flash-attn
```

Comment:
This shouldn't be here, as mentionned in the readme.
Also you only need one of those three, so adding the three is pointless